### DISCUSSION TITLE
Custom code

### DISCUSSION THREAD

**Discussion by LouayTech** [Impostor/Impostor#215] [Q&A] [answered]
Can i make custom codes?

**Comment by LorenzoPapi** [comment]
You mean create a room with a specific code?

**Comment by LouayTech** [comment]
> You mean create a room with a specific code?

Yes

**Comment by dimaguy** [comment]
No

**Comment by LorenzoPapi** [comment]
I don't think it's possible without modifying Impostor server itself, because the room code is a readonly variable

**Comment by Mm2PL** [comment, marked as answer]
Currently: no.
I think it might be possible by creating a room and then redirecting the player or making them reconnect. Doesn't look like it is easy to do though.

**Comment by LouayTech** [comment]
> Momenteel niet.
> Ik denk dat het mogelijk is door een kamer te maken en de speler vervolgens om te leiden of opnieuw verbinding te maken. Het ziet er echter niet naar uit dat het gemakkelijk te doen is.

Thanks for the answer
